docs/tutorials/8a_train_sqil_sac.ipynb

```diff
@@ -0,0 +1,160 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "[download this notebook here](https://github.com/HumanCompatibleAI/imitation/blob/master/docs/tutorials/8a_train_sqil_sac.ipynb)\n",
+    "# Train an Agent using Soft Q Imitation Learning with SAC\n",
+    "\n",
+    "In the previous tutorial, we used Soft Q Imitation Learning ([SQIL](https://arxiv.org/abs/1905.11108)) on top of DQN base algorithm. In fact, SQIL can be combined with any off-policy algorithm from `stable_baselines3`. Here, we train a HalfCheetah agent using SQIL + SAC."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "First, we need some expert trajectories in our environment (`seals/HalfCheetah-v0`).\n",
+    "Note that you can use other environments, but the action space must be continuous."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import datasets\n",
+    "from imitation.data import huggingface_utils\n",
+    "\n",
+    "# Download some expert trajectories from the HuggingFace Datasets Hub.\n",
+    "dataset = datasets.load_dataset(\"HumanCompatibleAI/ppo-seals-HalfCheetah-v0\")\n",
+    "\n",
+    "# Convert the dataset to a format usable by the imitation library.\n",
+    "expert_trajectories = huggingface_utils.TrajectoryDatasetSequence(dataset[\"train\"])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Let's quickly check if the expert is any good."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from imitation.data import rollout\n",
+    "\n",
+    "trajectory_stats = rollout.rollout_stats(expert_trajectories)\n",
+    "\n",
+    "print(\n",
+    "    f\"We have {trajectory_stats['n_traj']} trajectories. \"\n",
+    "    f\"The average length of each trajectory is {trajectory_stats['len_mean']}. \"\n",
+    "    f\"The average return of each trajectory is {trajectory_stats['return_mean']}.\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "After we collected our expert trajectories, it's time to set up our behavior cloning algorithm."
```

Comment:
I know this was just copied from the original tutorial but I find the reference to behavior cloning potentially ambiguous: it usually refers to supervised learning on expert trajectories (and we have a `BC` class that does exactly), SQIL is doing something conceptually similar but quite different in the details (RL rather than supervised learning).

Would suggest rephrasing this (and the original tutorial), could just call it an imitation algorithm rather than supervised learning algorithm.